src/Tvheadend.cpp

```diff
@@ -2530,6 +2545,34 @@ void CTvheadend::ParseRecordingAddOrUpdate(htsmsg_t* msg, bool bAdd)
   if (str)
     rec.SetFanartImage(GetImageURL(str));
 
+  uint32_t ageRating = 0;
+  if (!htsmsg_get_u32(msg, "ageRating", &ageRating))
+    rec.SetAgeRating(ageRating);
+
+  str = htsmsg_get_str(msg, "ratingLabel");
+  if (str)
+    rec.SetRatingLabel(str);
+  
+  str = htsmsg_get_str(msg, "ratingIcon");
+  if (str)
+    rec.SetRatingIcon(GetImageURL(str));
+
+  str = htsmsg_get_str(msg, "ratingAuthority");
+  if (str)
+  {
+    rec.SetRatingSource(str);
+  }
+  else
+  {
+    str = htsmsg_get_str(msg, "ratingCountry");
+    if (str)
+    {
+      rec.SetRatingSource(str);
+    }
+  }
+    
+
+  
```

Comment:
Can you please remove two of the three empty lines.

Reply:
Done.